### DISCUSSION TITLE
学习Vue源码求组队

### DISCUSSION THREAD

**Discussion by shengxinjing** [shengxinjing/it-roadmap#4] [通用]
学习Vue源码求组队

**Comment by QrnKG** [comment]
组起

**Reply by shengxinjing** [reply]
可以先从mini-vue看起

**Reply by chengjunjian2020** [reply]
我看的黄老师的之前只看到了响应式，watch还没看，过几天又把剩下的看起

**Comment by wakaka378** [comment]
带我一个

**Reply by wakaka378** [reply]
推荐一些个人学习Vue源码的技术文档和书籍：
- [Vue.js技术揭秘](https://ustbhuangyi.github.io/vue-analysis/)
- [Vue源码系列](https://vue-js.com/learn-vue/)
- [深入浅出Vue.js](https://book.douban.com/subject/32581281/)

**Comment by Link-CYD** [comment]
我个人发起了一个Vue2.6的源码项目, 目前读到了模板编译parse部分, 我希望有人一起讨论. 欢迎联系我😊  

项目👉https://github.com/LinkSofuny/vue-core-analyse

**Comment by ddddderek** [comment]
+1
